### PR TITLE
Feature/new poses

### PR DESCRIPTION
**Pull request by RaionArt**
Adds 3 new poses and all needed sleeves:
- `finger_touching`
- `pointy_finger`
- `hand_on_chin`

### CODE REVIEW

File: game/sprite-backend.rpy [hunk 1]
```diff
@@ -1673,7 +1682,10 @@ label pose_test:
...
     n 2fllpo "This is my arms_crossed_body pose!"
     n 3kwmsml "This is my arms_crossed_desk pose!"
-    n 4kwmpu "And this is my fingers_on_desk pose!"
+    n 4kwmpu "This is my fingers_on_desk pose!"
+    n 5kslajl "This is my finger_touching pose!"
+    n 6fcsss "This is my pointy_finger pose!"
+    n 7ttlbo "And this is my hand_on_chin pose!"
 
     n 1fchbg "...And that's about it!{w=0.75}{nw}"
     extend 1fwlsm  " Glad to be of service,{w=0.2} [player]!"
```
Comment (Blizzardsev) [inline game/sprite-backend.rpy:1691]: We probably shouldn't be including test code! Can you get rid of this dialogue?

File: game/sprite-backend.rpy [hunk 2]
```diff
@@ -170,13 +173,16 @@ init -50 python:
...
     ]
 
-    # These are poses with arms rendere on top of the desk
+    # These are poses with arms rendered on top of the desk
     _JN_AFTER_DESK_POSES = [
         JNPose.arms_crossed_desk,
-        JNPose.fingers_on_desk
+        JNPose.fingers_on_desk,
+        JNPose.pointy_finger,
+        JNPose.hand_on_chin
```
Comment (Blizzardsev) [inline game/sprite-backend.rpy:185]: Good use of the split between the before/after desk poses - well done!

File: game/mod_assets/natsuki/clothes/jn_clothes_nya_sweater/fingers_on_desk.png
Comment (Blizzardsev) [inline game/mod_assets/natsuki/clothes/jn_clothes_nya_sweater/fingers_on_desk.png:1]: I'm a little confused here - I can see that some versions of this clothing have been replaced with an adjusted version that hugs Natsuki's body better, but not all of them. Should they be consistent?

**Review by Blizzardsev — CHANGES_REQUESTED** [review]
Just a couple of things I spotted, but the major code additions to support this are in the clear! Big well done for all the work that went into the sprites: make sure you get some rest!

File: game/mod_assets/natsuki/clothes/jn_clothes_nya_sweater/fingers_on_desk.png
Comment (Blizzardsev) [reply game/mod_assets/natsuki/clothes/jn_clothes_nya_sweater/fingers_on_desk.png:1]: Might be worth seeing if any of the other sweaters have the same problem!

Comment (RaionArt) [reply game/mod_assets/natsuki/clothes/jn_clothes_nya_sweater/fingers_on_desk.png:1]: I only changed the nya_sweater as Nemu didn't adjust it while changing to the new body of Natsuki, good catch! But there is no need to check the other sweaters as it was an exception for this one.

Comment (Blizzardsev) [reply game/mod_assets/natsuki/clothes/jn_clothes_nya_sweater/fingers_on_desk.png:1]: No worries!

**Review by Blizzardsev — APPROVED** [review]
Great work - can't wait to begin rolling these out! Now get some rest!

**Pull request merged**